include/sound/sof/xman.h

```diff
@@ -39,6 +39,7 @@ struct xman_header {
 
 /* Extended manifest elements identificators */
 enum xman_elem_type {
+	XMAN_ELEM_FW_VERSION	= 1,
```

Comment:
fix your commit messages please....This is going to be reviewed by maintainers, this is not optional or to be taken lightly.

Reply:
changed